src/transformers/models/biogpt/configuration_biogpt.py

```diff
@@ -111,7 +111,6 @@ def __init__(
         layer_norm_eps=1e-12,
         scale_embedding=True,
         use_cache=True,
-        is_encoder_decoder=False,
```

Comment:
`is_encoder_decoder` may not appear in modeling file, but it's important in generation files.
The default value in `PretrainedConfig` is `False`. Let's not add it here to make thing a bit easier/cleaner.